lms/djangoapps/certificates/views/webview.py

```diff
@@ -416,10 +417,12 @@ def render_html_view(request, user_id, course_id):
         )
     )
     context['share_url'] = share_url
-    twitter_url = 'https://twitter.com/intent/tweet?text={twitter_share_text}&url={share_url}'.format(
-        twitter_share_text=context['twitter_share_text'],
-        share_url=urllib.quote_plus(share_url)
-    )
+    twitter_url = ''
```

Comment:
Please explain our use of smart_str for this particular area as a code comment, because it might not be clear to future contributors why it has been included here.